### PR TITLE
freexl 1.0.2

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

File: Library/Formula/freexl.rb [hunk 1]
```diff
@@ -27,4 +27,8 @@ def install
       doc.install "html"
     end
   end
+
+  test do
+    assert File.exist? "#{lib}/libfreexl.a"
```
Comment (xu-cheng) [inline Library/Formula/freexl.rb]: I think we should avoid check file existing in the test. They are basically meaningless.

Comment (nijikon) [inline Library/Formula/freexl.rb]: How we should test this then?

Comment (xu-cheng) [inline Library/Formula/freexl.rb]: Can you write a simple C program? You can see `tinyxml2` for example.

Comment (nijikon) [inline Library/Formula/freexl.rb]: No problem, so we would also have to fix some more packages that have this kind of tests.

Comment (xu-cheng) [inline Library/Formula/freexl.rb]: >  so we would also have to fix some more packages that have this kind of tests.

That would be extremely appreciated.

**Comment by DomT4** [comment]
@BrewTestBot test this please

**Comment by DomT4** [comment]
Presuming it passes again I'll merge it & remove the `exist` test here. The version bump is something that we try not to hold up too much, but if you wanted to chase in the merge with a new PR for a new, shiny test I certainly wouldn't *object* :wink:.

**Comment by DomT4** [comment]
I meant to remove the test prior to merge, rather than after, but I'm tired and apparently an idiot :smiley:.

Merged in https://github.com/Homebrew/homebrew/commit/79a3233459efe1f48b8c61d6b19833e14f087d3a. Thanks @nijikon! :)

**Pull request closed without merging**